src/main/java/io/netty/buffer/api/CompositeBuf.java

```diff
@@ -129,6 +130,7 @@ public String toString() {
 
     @Override
     public Buf order(ByteOrder order) {
+        this.order = order;
```

Comment:
nit: should we check that the order is actually changed and if not just return early ?